doc/src/sgml/installation-meson.sgml

```diff
@@ -0,0 +1,2635 @@
+<!-- doc/src/sgml/installation-meson.sgml -->
+<!--
+
+The standalone version has some portions that are different from the version
+that is integrated into the full documentation set, in particular as regards
+links, so that INSTALL.html can be created without links to the main
+documentation.  See standalone-profile.xsl for details.
+
+-->
+
+<chapter id="installation-meson">
+ <title>Installation from Source Code with Meson</title>
+
+ <indexterm zone="installation">
+  <primary>installation</primary>
+ </indexterm>
+
+ <!-- See also the version of this text in standalone-install.xml -->
+ <para>
+  This chapter describes the installation of
+  <productname>PostgreSQL</productname> using the source code
+  distribution.  If you are installing a pre-packaged distribution,
+  such as an RPM or Debian package, ignore this chapter
+  and see <xref linkend="install-binaries" /> instead.
+ </para>
+
+ <para>
+  What is Meson and Why Meson?
+ </para>
+
+ <para>
+  Comparison of 2 build systems
+ </para>
+
+ <para>
+  If you are building <productname>PostgreSQL</productname> for Microsoft
+  Windows, read this chapter if you intend to build with MinGW or Cygwin;
+  but if you intend to build with Microsoft's <productname>Visual
+  C++</productname>, see <xref linkend="install-windows"/> instead.
+ </para>
+
+ <sect1 id="install-short-meson">
+  <title>Short Version</title>
+
+  <para>
+<synopsis>
+# fetch source code
+# git clone https://github.com/postgres/postgres.git
+# cd postgres
+
+# current instructions for testing (to be removed when merging)
+git clone https://github.com/anarazel/postgres.git
+cd postgres
+git switch meson
+
+# setup and enter build directory (done only first time)
+# meson setup build
+meson setup build --prefix=$PWD/install
```

Comment:
Using $PWD has the disadvantage of not working on windows. Wonder if we can come up with something better.